contrib/learn_resource_weights.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import json
+import logging
+
+logging.basicConfig(format='[%(levelname)s] %(message)s')
+
+
+def parse_commandline():
+    """Parse commandline arguments"""
+    import argparse
+    parser = argparse.ArgumentParser(description='export and analyze resources from statistics',
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('command', choices=[
+                        'parse', 'analyze'], help='path of benchmark results')
+    parser.add_argument('basedir', default=None, nargs='?',
+                        help='path of benchmark results')
+    parser.add_argument('-v', '--verbose',
+                        action='store_true', help='be more verbose')
+    parser.add_argument('--json', default='data.json.gz',
+                        help='path of json file')
+    parser.add_argument('--threshold', metavar='SEC', type=int, default=1,
+                        help='ignore benchmarks with a runtime below this threshold')
+    parser.add_argument('--mult', type=int, default=1000,
+                        help='multiply running times with this factor for regression')
+
+    return parser.parse_args()
+
+
+args = parse_commandline()
+
+if args.verbose:
+    logging.getLogger().setLevel(level=logging.DEBUG)
+else:
+    logging.getLogger().setLevel(level=logging.INFO)
+
+
+def load_zipped_json(filename):
+    """Load data from a gziped json file"""
+    import gzip
+    with gzip.GzipFile(args.json, 'r') as fin:
+        return json.loads(fin.read().decode('utf-8'))
+
+
+def save_zipped_json(filename, data):
+    """Store data to a gziped json file"""
+    import gzip
+    with gzip.GzipFile(args.json, 'w') as fout:
+        fout.write(json.dumps(data).encode('utf-8'))
+
+
+def get_sorted_values(data):
+    """Transform [['name', value], ...] to [value, ...] sorted by names"""
+    return [d[1] for d in sorted(data)]
+
+
+if args.command == 'parse':
+    if args.basedir == None:
+        raise Exception('Specify basedir for parsing!')
+    import glob
+    import re
+    filename_re = re.compile(
+        '.*_incremental_([A-Z_]+/[0-9a-zA-Z_=\\./-]+\\.smt2)/output\\.log')
```

Comment:
Why not simply:
```suggestion
        '.*\\.smt2)/output\\.log')
```
to make sure that it also works on custom benchmark sets.